libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -495,16 +491,16 @@ bool ContractCompiler::visit(FunctionDefinition const& _function)
 		}
 	//@todo assert that everything is in place now
```

Comment:
Perhaps we should actually implement this now.
This would be a check that the elements in `stackLayout` are `0,1,2,....,k`